plugins/isp/internode_usage

```diff
@@ -221,14 +221,14 @@ graph_data() {
 	echo "multigraph internode_usage_${SERVICE_ID}${graph}"
 	case "${graph}" in
 		.current)
-			echo "hourly_rate.value ${SERVICE_USAGE:-U}"
+			echo "hourly_rate.value ${CURRENT_TIMESTAMP}:${SERVICE_USAGE:-U}"
 			;;
 		.daily)
 			echo "daily_rate.value ${DAILY_TIMESTAMP}:${DAILY_USAGE:-U}"
 			;;
 		*)
-			echo "usage.value ${SERVICE_USAGE:-U}"
-			echo "ideal.value ${IDEAL_USAGE:-U}"
+			echo "usage.value ${CURRENT_TIMESTAMP}:${SERVICE_USAGE:-U}"
```

Comment:
Is it really necessary to add the timestamp here? This sounds like a bug in the spool processing to me.

Reply:
It is, but there's not real way around it \:

Reply:
If it is a bug: where is the ticket? (I would like to solve it instead of fostering bad examples in plugin code).

Reply:
Actually, maybe it's not a bug as such, but a corner case.

This particular plugin outputs data with two different dates: right now, and more or less a day before. Munin is not used to it, and assumes that all the data from a plugin comes from the same time, which is not the case here. Not stating the timestamp explicitely here leads munin to assume that it's the same as the delayed one, which is often lower than the last received statement, which leads Munin to discard fresh values from the current time.

I think it's more of a weird interaction between Munin's expectations, and the data from the API this plugin queries.